docs/source/host/server-setup/linux-server.md

```diff
@@ -16,7 +16,7 @@ sudo apt-get update && sudo apt-get install ./vircadia-server_2020.2.0-asteria-2
 ### Amazon Linux 2
 
 ```sh
-sudo yum install https://cdn.vircadia.com/dist/domain-server/amazon-linux/athena-server-0.86.0_K2_20200402_76aff7a-1.amzn2.x86_64.rpm
+sudo yum install https://cdn.vircadia.com/dist/domain-server/amazon-linux/vircadia-server-2020.2.0_ASTERIA_20200608_585fb19-1.amzn2.x86_64.rpm
```

Comment:
This is still 2020.2.0.

Reply:
These changes don't change anything for the server particularly so that's still fine. :)